boilerplate/registry/index.html

```diff
@@ -0,0 +1,288 @@
+          ttwg: {
+            title: "Timed Text Working Group",
+            href: "https://www.w3.org/AudioVideo/TT/"
+          }
+        }
+      };
+    </script>
+  </head>
+  <body>
+    <section id="abstract">
+      <p>This is a boilerplate Registry Definition as required by [[w3c-process]] for [=w3c registries=].</p>
+      <p>The intent is that this boilerplate can form the basis of [=w3c registries=] for
+        the W3C <dfn data-abbr="TTWG"><a href="https://www.w3.org/AudioVideo/TT/">Timed Text Working Group</a></dfn>.</p>
+      <p>See discussion <a href="https://github.com/w3c/ttwg/discussions/241">w3c/ttwg#241</a> for
+        details of the assumptions made to generate this document.</p>
+    </section>
+    <section id="sotd">
+      <p>This document is intended to be the basis of other [=w3c registry=] documents
+        published by TTWG and is not intended to be published as a [=w3c registry=] itself.
+        The following SOTD text is auto-generated by the Respec tool to reflect
+        what an actual <a>registry definition</a> document would contain.</p>
+    </section>
+    <section class="informative">
+      <h2>Introduction</h2>
+      <p>Replace the text in this section with some
+        informative introductory text about this Registry.</p>
+      <p>This document is intended to be used as the basis for [=w3c registries=] for
+        the W3C <a>TTWG</a>.
+        The [[w3c-process]] sets out requirements for such [=w3c registries=],
+        and defines core concepts like
+        <dfn data-cite="w3c-process#registry-definition">registry definition</dfn>
+        and <dfn data-cite="w3c-process#registry-table">registry table</dfn>.</p>
+      <p><a>Registry definitions</a> need to fulfil certain criteria,
+        including specifying custodianship, the change process,
+        and defining the <a>registry table</a> itself.</p>
+      <p>The text in the <a href="#registry-definition"></a> section can be
+        used to meet those criteria and allow more rapid creation of
+        registries without having to create new text each time.</p>
+      <p>This document also includes appendix <a href="#application-checklist"></a>
+        to facilitate adoption.</p>
+    </section>
+    <section id="registry-definition">
+      <h2>Registry Definition</h2>
+      <p>This section specifies the <a>registry definition</a>, consisting of
+        the custodianship, change process and
+        the core requirements of a <a>registry table</a>.
+      </p>
+      <section id="custodianship">
+        <h3>Custodianship</h3>
+        <p>The <dfn data-cite="w3c-process#custodian">custodian</dfn> of this [=w3c registry=] is the <a>TTWG</a>. 
+          If the <a>TTWG</a> is unable to fulfil the role of <a>custodian</a>,
+          for example if it has been <a data-cite="w3c-process#GeneralTermination">closed</a>,
+          the <a>custodian</a> in lieu is the <a data-cite="w3c-process#Team">W3C Team</a>.
```

Comment:
Do we have some strong reason to link a section in the Process (`#Team`), but not a defined term (`#team`)? (this makes duplicate-liked entry in section A.2....)
(Note: duplicate-liked entry should be fine, but I may need to explain why at the point of publication...)

Reply:
Not sure, but I've fixed it in ed31515. Also I realised I was using inconsistent markup for references, and have adopted `<a>ref</a>` throughout in place of `[=ref=]`.